content/docs/1.7.0/snapshots-and-backups/csi-volume-clone.md

```diff
@@ -41,8 +43,35 @@ spec:
 > In addition to the requirements listed at [CSI volume cloning](https://kubernetes.io/docs/concepts/storage/volume-pvc-datasource/),
 > the `cloned-pvc` must have the same `resources.requests.storage` as the `source-pvc`.
 
+
+## Via Longhorn UI
+
+To clone volume from existing volume
+1. Go to volume page
+2. Choose a volume and click **Clone Volume** from operation menu
+3. You can edit the cloned volume configurations and click Ok.
+
+To clone volume from volume snapshot.
+1. Go to volume page, click volume name to enter volume detail page
+2. In **Snapshot and Backups** block, click *Clone Volume* from any snapshot or backup
+3. You can edit the cloned volume configurations and click Ok.
```

Comment:
```suggestion
### Clone a Volume Using a Snapshot

1. Go to the **Volume** screen.

1. Click the name of the volume that you want to clone.

1. In the **Snapshot and Backups** section of the details screen, identify the snapshot that you want to use and then click **Clone Volume**.

1. (Optional) Configure the settings of the new volume.

1. Click **OK**.
```

Reply:
> 2. In **Snapshot and Backups** block, click *Clone Volume* from any snapshot or backup

I don't know this means exactly because the screenshot does not show it. I'm guessing you are referring to a section that lists available snapshots and backups.

Reply:
yeah, the available action when click snapshot or backup.
<img width="1428" alt="image" src="https://github.com/longhorn/website/assets/5744158/a398d39e-335d-495b-a38f-161c6fecd9dc">

Reply:
Thank you for the screenshot. The suggested wording should work for the UI flow.